docs/getting-started/grouped-nav-ui.asciidoc

```diff
@@ -1,16 +1,16 @@
 [[grouped-nav-ui]]
 = Grouped navigation menu
 
-{elastic-sec}'s grouped navigation menu groups related pages and highlights commonly visited areas for a streamlined experience. 
+{elastic-sec}'s grouped navigation menu groups related pages and highlights commonly visited areas for a streamlined experience.
 
-NOTE: By default, the {security-app} uses the legacy navigation menu. To enable grouped navigation, go to *{kib}* -> *{stack-manage-app}* -> *Advanced Settings*, then turn on `securitySolution:enableGroupedNav`.
+NOTE: By default, the {security-app} uses the grouped navigation menu. To turn on legacy navigation, go to *{kib}* -> *{stack-manage-app}* -> *Advanced Settings*, then turn off `securitySolution:enableGroupedNav`.
```

Comment:
This topic will actually end up being deleted since the grouped nav will be on default, but I'll be taking the pieces from this topic to the Elastic Security UI one - just FYI.